ogre2/src/Ogre2Material.cc

```diff
@@ -87,6 +89,38 @@ void Ogre2Material::Destroy()
     matManager.remove(this->ogreMaterial);
     this->ogreMaterial.reset();
   }
+
+  auto &textureManager = Ogre::TextureManager::getSingleton();
+  auto iend = textureManager.getResourceIterator().end();
+  for (auto i = textureManager.getResourceIterator().begin(); i != iend;)
+  {
+    // A use count of 4 means that only RGM, RM and MeshManager have references
+    // RGM has one (this one) and RM has 2 (by name and by handle)
+    // and MeshManager keep another one int the template
+    Ogre::Resource* res = i->second.get();
+    if (i->second.useCount() == 5)
+    {
+      if (this->dataPtr->hashName == res->getName() &&
+          res->getName().find("scene::RenderTexture") == std::string::npos)
```

Comment:
`scene::RenderTexture` -> the prefix `scene` is the name of the scene so how about changing this to `scene->Name() + "::RenderTexture"`